src/Microsoft.Diagnostics.EventFlow.Outputs.ElasticSearch/ElasticSearchOutput.cs

```diff
@@ -388,6 +388,7 @@ private async Task EnsureIndexExists(string indexName, ElasticClient esClient)
             indexSettings.Settings.NumberOfReplicas = this.connectionData.Configuration.NumberOfReplicas;
             indexSettings.Settings.NumberOfShards = this.connectionData.Configuration.NumberOfShards;
             indexSettings.Settings.Add("refresh_interval", this.connectionData.Configuration.RefreshInterval);
+            indexSettings.Settings.Add("default_pipeline", this.connectionData.Configuration.DefaultPipeline);
```

Comment:
Should we only set it if `Configuration.DefaultPipeline` is not null or empty?

Reply:
I thought about that, but I did a quick test of the Nest IndexSettings class and serializer, and found that if the dictionary entry had a null value, the field did not get rendered in the output, which I then confirmed with another quick test against a test elastic instance. 

It wouldn't hurt anything to wrap it in an if statement, I was just going for minimal additional code.

And thank you guys for this framework! It has been very helpful.

Reply:
Thank you!

Yes, makes sense, and I am glad that Nest is doing the right thing here, but let's be a bit more defensive in case they "change their mind" down the road ☺️

Reply:
Added another commit with a null check, such that this parameter will only be added to the index settings if the corresponding Configuration property has a non-null value.